packages/plugins/ganache/src/index.js

```diff
@@ -1,10 +1,41 @@
+const constants = require('embark-core/constants');
+
 class Ganache {
   constructor(embark) {
-    embark.events.request('blockchain:vm:register', () => {
+    embark.events.request("blockchain:node:register", constants.blockchain.clients.ganache, {
+      isStartedFn: (cb) => {
+        cb(null, true); // Always assume it's started, because it's just a provider (nothing to start)
+      },
+      launchFn: (cb) => {
+        cb(); // Will probably never be called, but if so, callback straight aways because there is nothing to start
+      },
+      stopFn: async (cb) => {
```

Comment:
I do believe this gets called by the tests by way of the event `blockchain:node:stop`.

Reply:
same as above